frontend/arthas-ai/app/components/Navbar.tsx

```diff
@@ -0,0 +1,90 @@
+import { BoxIcon } from 'lucide-react';
```

Comment:
add `'use client'` to the top as always. It might throw an error since every react component is serverside by default